src/condition-check/condition-check.ts

```diff
@@ -14,21 +14,29 @@ export const calculateUpdateInPercentage = (initialValue: ethers.BigNumber, upda
 
 export const calculateMedian = (arr: ethers.BigNumber[]) => {
   const mid = Math.floor(arr.length / 2);
+
   const nums = [...arr].sort((a, b) => {
     if (a.lt(b)) return -1;
     else if (a.gt(b)) return 1;
     else return 0;
   });
 
-  if (arr.length % 2 !== 0) {
-    return nums[mid];
-  }
+  const midNumber = nums[mid];
```

Comment:
Can't you just put some `!` ? I don't think any of these errors can actually happen.